superannotate/db/project_images.py

```diff
@@ -634,25 +614,24 @@ def assign_folder(project, folder_name, users):
     """
 
     project_meta = get_project_metadata(project, include_contributors=True)
-    project_users = project_meta["contributors"]
+    project_users = get_team_metadata()["users"]
     project_name = project_meta['name']
-    project_users = [i['user_id'] for i in project_users]
-    verified_contributor = []
+    project_users = [i['id'] for i in project_users]
+    unverified_contributor = set(project_users) - set(users)
```

Comment:
Wrong statement, should be users - project_users

Reply:
@davitavagyan updated.